### ISSUE TITLE
请问有没有SROIE数据集文本检测和识别训练脚本

### ISSUE THREAD

**Issue by Andrew-Zhu** [open-mmlab/mmocr#1114]
如题，似乎SROIE数据集没有提供文本检测和文本识别训练脚本，请问能够尽快提供呢？或者提供相关参考资料帮助我们在此数据集上开发呢？

**Comment by Andrew-Zhu** [comment]
Please provide training scripts for text detection and recognition in the SROIE dataset

**Comment by gaotongxiao** [comment]
Hi, currently we have provided the conversion script for SROIE for text detection & recognition tasks. With the converted annotations, you can customize your base dataset configs in the similar way as https://github.com/open-mmlab/mmocr/blob/main/configs/_base_/det_datasets/icdar2015.py and https://github.com/open-mmlab/mmocr/blob/main/configs/_base_/recog_datasets/MJ_train.py, then import these base configs in your model configs to start the training.

**Comment by Andrew-Zhu** [comment]
Actually, I want to train the model on SROIE. However, I do not have any knowledge about this dataset. For example, I need a suitable learning rate for this dataset.
